xla/tools/hlo_opt/tests/run_algsimp_pass.hlo

```diff
@@ -0,0 +1,15 @@
+// NOTE: Assertions have been autogenerated by hlo/tools/generate_hlo_test_checks.py
```

Comment:
This is missing a RUN: line. Please check the other test as reference how it needs to look like.